packages/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -163,20 +163,105 @@ import { Chart, ChartBar, ChartVoronoiContainer } from '@patternfly/react-charts
 </div>
 ```
 
+### Basic with right aligned legend (Accessibility Demo)
+
+This demo demonstrate the accessibility function in a Bar chart. You can now focus on the whole chart or the contents in the chart. At the same time, you could use screen reader like VoiceOver to read the chart and contents as you defined in format.
+
```

Comment:
I believe this demo would be best in its own a11y section? Similar to functionality for tooltips, legend, etc. We could build more complex examples there.

Reply:
Will do. Thanks!